frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -299,7 +299,7 @@ describe('CRD extensions', () => {
       await yamlView.setEditorContent(safeDump(newContent));
       expect(yamlView.getEditorContent()).toContain(`namespaceFilter: ${namespaceFilter}`);
       await yamlView.saveButton.click();
-      await browser.wait(until.visibilityOf(crudView.successMessage), 1000);
+      await browser.wait(until.visibilityOf(crudView.successMessage), BROWSER_TIMEOUT);
```

Comment:
Same question here.

Reply:
Dropped the 2nd param, and verified FF worked.